src/main/java/org/jenkinsci/plugins/ewm/actions/ExwsAllocateActionImpl.java

```diff
@@ -23,12 +26,14 @@
     private final List<ExternalWorkspace> allocatedWorkspaces = new LinkedList<>();
```

Comment:
@oleg-nenashev WDYT, should I replace this `List` with a `Map`?
(of course, not actually remove the field, but make it transient, and provide the `readResolve()` method)